cot/src/lib.rs

```diff
@@ -92,8 +92,8 @@ pub use body::Body;
 /// Cot-powered app.
 ///
 /// This is primarily useful for use with the
-/// [`TestServer`](cot::test::TestServer) struct, which allows you to run a
-/// full-fledged Cot server in a test environment.
+/// [`TestServer`](cot::test::TestServerBuilder) struct, which allows you to run
```

Comment:
I think you missed that one
```suggestion
/// [`TestServerBuilder`](cot::test::TestServerBuilder) struct, which allows you to run
```

Reply:
Right, thanks for catching this!